pipelines/e2e/scripts/longhorn-setup.sh

```diff
@@ -25,6 +25,16 @@ create_instance_mapping_configmap(){
   kubectl create configmap instance-mapping --from-file=/tmp/instance_mapping
 }
 
+longhornctl_check(){
+  curl -L https://github.com/longhorn/cli/releases/download/v1.7.0-rc2/longhornctl-linux-amd64 -o longhornctl
```

Comment:
We could use `v1.7.1-rc2` from https://github.com/longhorn/cli/releases/download/v1.7.1-rc2/longhornctl-linux-amd64 for now.

Reply:
Updated. Thank you!